app/src/main/java/chat/rocket/android/util/SerialisationUtils.kt

```diff
@@ -0,0 +1,22 @@
+package chat.rocket.android.util
+
+import timber.log.Timber
+import java.io.*
+
+var byteArray: ByteArray? = null
+
+fun serialiseToken(token: TokenSerialisableModel): ByteArray? {
+    val a = ByteArrayOutputStream()
+    val b = ObjectOutputStream(a)
+    try {
+        b.writeObject(token)
+        byteArray = a.toByteArray()
+    } catch (ex: InvalidClassException) {
```

Comment:
To avoid multiples `catch` here you can use only one `catch` with the more generic `Exception`.

Reply:
I added specific catch messages for easy debugging.